locations/c-base.yml

```diff
@@ -0,0 +1,96 @@
+---
+location: c-base
+location_nice: "Rungestraße 20, 10179 Berlin"
+latitude: 52.513451
```

Comment:
Coordinates seem to be off. The location was shown in the water: https://hopglass.berlin.freifunk.net/#!v:m;n:c-base-core.olsr

Reply:
should be set to 52.512865,13.42017

Reply:
resolved by [d109212](https://github.com/freifunk-berlin/bbb-configs/pull/963/commits/d109212878319ddab1de45ff8453fa6c8383fbea)